packages/devtools/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
-## 0.0.1
+## 0.0.14 - 2019-03-15
+* Support dark mode.  Add ```&theme=dark``` at the end of URI used to open DevTools in the Chrome browser.
+### Memory
+* Added showing GCs on the timeline and leak detection.
+### Timeline
+* Handle frames with delay.
```

Comment:
Change this one from what the fix was  to what the feature aded was
Feature is
* Fix bugs when events were received out of order.

Reply:
Done.